SDC-seed/PostgreSQL/listingCSVMaker.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+
+const writeListings = fs.createWriteStream('listingTable.csv');
+writeListings.write('listingId,reviews,avatar\n', 'utf8');
```

Comment:
Also on line 4, your header line might get overwritten by the subsequent writing of data in your do-while loop. I think a .write() overwrites whatever is in the targeted file path instead of appending. You could add a conditional statement inside your do-while loop below, that on the 10,000,001-th pass (i.e. the first pass) to write the header line instead of a line of data.